src/promptflow/promptflow/_sdk/_configuration.py

```diff
@@ -177,6 +179,14 @@ def is_eu_user(self) -> Optional[bool]:
             return self.get_config(key=self.EXTENSION_EU_USER)
         return self.get_config(key=self.EU_USER)
 
+    def get_recording_mode(self) -> Optional[str]:
+        """Check if recording mode is enabled."""
+        return self.get_config(key=self.RECORDING_MODE)
```

Comment:
Shall there be any default mode? default to live?

Reply:
Yes, let me make some refinement.